specs/marketplace.md

```diff
@@ -185,178 +174,159 @@ struct Content {
 
 ```
 
-Notes about some parameters:
+The meaning of the Request parameters is given below:
```

Comment:
This wouldn't be needed if we put the description for all properties below the code block.

Reply:
https://github.com/codex-storage/codex-spec/pull/3#discussion_r1728859064